### DISCUSSION TITLE
Show and Hide Annotations

### DISCUSSION THREAD

**Discussion by andrheapaltao** [annotorious/annotorious#261] [Q&A] [answered]
Hello, I'm writing a function that shows and hides the box of the annotations. Hiding the annotation using the `removeAnnotation` works and my question is: What should I use to return the box of the removed annotation? Is it by using `getAnnotationById`?

**Comment by rsimon** [comment, marked as answer]
If you remove the annotation, it's completely removed from Annotorious. To show it again, you would have to: 

- Keep a copy of the annotation in a variable. You can use `getAnnotationById` to get the annotation. However this will only work __before__ you remove it.
- re-add it to Annotorious, using `addAnnotation`, to show it again

**Reply by andrheapaltao** [reply]
I see. Thank you so much for that!
